.github/workflows/deploy-eb.yml

```diff
@@ -0,0 +1,40 @@
+name: Deploy EB Prod
+
+on:
+  push:
+    branches:
+      - master
+  repository_dispatch:
+    types: manual-trigger
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+    - name: Login to DockerHub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_PASSWORD }}
+    - name: Build and push docker image
+      id: docker_build
+      uses: docker/build-push-action@v2
+      with:
+        push: true
+        tags: serotracker/iit-backend:latest
+        file: Dockerfile-app
+    - name: Setup Python
+      uses: actions/setup-python@v2
```

Comment:
should this be setup-python@v2 cause we're using version 3? not sure if it's related to python version

Reply:
Not related, see line 31 where we specify python version. This @v2 refers to v2 of this github actions plugin